Mathlib/Algebra/Module/End.lean

```diff
@@ -57,9 +57,14 @@ def smulAddHom : R →+ M →+ M :=
 variable {R M}
 
 @[simp]
-theorem smulAddHom_apply (r : R) (x : M) : smulAddHom R M r x = r • x :=
+theorem smulAddHom_apply : smulAddHom R M r x = r • x :=
   rfl
 
+variable {r}
+
+lemma IsAddUnit.smul (hr : IsAddUnit r) : IsAddUnit (r • x) :=
```

Comment:
I think I'd expect this to be `smul_const`, to disambiguate from `const_smul` which I assume also holds?